iconset.json

```diff
@@ -5,17 +5,17 @@
     {
       "name": "Unsorted",
       "properties": {
-        "bgCircle": false,
-        "bgFill": "#000",
-        "bgRadius": 4,
-        "bgShape": false,
+        "fill": "#725a50",
         "bgSize": 4,
-        "bgStroke": "#000",
+        "bgFill": "#000",
         "bgStrokeWidth": 0,
-        "fill": "#725a50",
+        "strokeWidth": 0,
+        "bgCircle": false,
+        "bgRadius": 4,
```

Comment:
It would be better not to commit this resorting, even if `maki` did this. You can soft reset via git and then selectively commit individual lines. After committing just reset hard.

In order that the above works you will need to remove the `git add` from the following line: https://github.com/digitalfabrik/integreat-maplibre-style/blob/gh-pages/package.json#L36
You can commit this change in a separate commit.